projects/ad2s1210_iio/.gitignore

```diff
@@ -0,0 +1,12 @@
+*.swp
```

Comment:
to be consistent with other projects, please use below content in this file:

*.swp
*.profile*
build
.vs/
.visualgdb/
*.sln
*.vgdbproj
*.TMP
*.user
tests/.pytest_cache
tests/__pycache__
tests/func/__pycache__
tests/output
tests/.idea